src/com/estafet/learning/US008/Order.java

```diff
@@ -0,0 +1,27 @@
+package com.estafet.learning.US008;
+
+public abstract class Order {
+    public static final String ORDER_NAME = "ORR";
+    public String OrderNumber;
+    public String ClientDetails;
```

Comment:
all field names should start with **lowerCase** letters as this looks like **Types**.

Reply:
changed.